HOWTO/SBOM.md

```diff
@@ -237,3 +241,399 @@ Delete the code and any remaining `vendor.info` files.
+
+### HOW-TO
+
+Erlang/OTP will maintain VEX files for the latests three releases.
+Because of this, Erlang/OTP will always contain the latest information in the `master` branch.
+Any OpenVEX file in other branches is considered outdated.
+
+The OpenVEX files are located in `vex/otp-26.openvex.json`, `vex/otp-27.openvex.json`, and `vex/otp-28.openvex.json` (e.g.). These files are generated from the `make/openvex.table` and the script `.github/scripts/otp-compliance.es`.
+
+- `make/openvex.table` contains all known CVEs on a per release basis, with top-level objects for `otp-XX` branches, where each `otp-XX` object has as value a list of dependencies with their CVE and the status.
+
+  Example:
+
+  ```json
+      "otp-28":
+      [
+          {
+              "pkg:github/openssl/openssl@636dfadc70ce26f2473870570bfd9ec352806b1d" : "CVE-2025-4575",
+              "status": {"not_affected": "vulnerable_code_not_present"}
+          },
+
+          {
+              "pkg:github/PCRE2Project/pcre2@2dce7761b1831fd3f82a9c2bd5476259d945da4d": "OSV-2025-300",
+              "status": {"not_affected": "vulnerable_code_not_present"}
+          },
+          ...
+      ]
+  ```
+
+The `status` corresponds to the possible status from the [OpenVEX specification](https://github.com/openvex/spec/blob/main/OPENVEX-SPEC.md).
+In case of `not_affected`, a reason must be provided (similar to the [specification](https://github.com/openvex/spec/blob/main/OPENVEX-SPEC.md)).
+**The `make/openvex.table` is considered to be an append-only structure, where one should not do modifications to existing data nor removal**.
+Changes should be done via `.github/scripts/otp-compliance.es` applied on the `openvex.table`. The main reason is to use
+`openvex.table` as a simple source of truth without boilerplate, since VEX statements can be long due to the way in which one
+must express range versions for a vulnerability.
+
+### Further Format Details of openvex.table
+
+The file `openvex.table` is a subset of fields of the OpenVEX specification.
+The format is a JSON object that contains OTP VEX statements. A top-level valid field is the
+OTP version key (e.g., `otp-29`), followed by a list of objects. Each JSON object can have the following structure.
+
+- A key with a Purl, which uniquely identifies the application that the statement talks about,
+  and a CVE string as value.
+- A key `status` with value `Status :: "affected" | "fixed" | "under_investigation | "not_affected" | Affected`,
+  where `Affected` is an object explained below.
+- `Affected` is an object that may have the following keys
+  - `affected` with value string that explains mitigation strategies
+  - `fixed` with value where the fix was introduced.
+
+**Example**
+
+```json
+{
+  "otp-23": [
+    {
+      "pkg:otp/ssl@10.2.1": "CVE-2025-26618",
+      "status": { "affected": "Update to the next version",
+                  "fixed": ["pkg:otp/ssl@10.3.1.1", "pkg:otp/ssl@10.2.4.2"]}
```

Comment:
If you are going to use a real CVE here you should use the real fixes with the correct application I think.  This CVE was for ssh not ssl.  https://github.com/erlang/otp/security/advisories/GHSA-78cv-45vx-q6fr